src/metabase/core.clj

```diff
@@ -44,10 +44,9 @@
 
 ;;; --------------------------------------------------- Lifecycle ----------------------------------------------------
 
-(defn- -init-create-setup-token
-  "Create and set a new setup token and log it."
+(defn- print-setup-url
+  "Used to print the setup url during instance initialization."
```

Comment:
"Used to" adds nothing here. "Print the setup url during instance initialization" is the same but without being verbose.